cli/src/semgrep/formatter/text.py

```diff
@@ -99,6 +99,8 @@ def _finding_to_line(
             dedented_lines = textwrap.dedent("".join(lines)).splitlines()
             indent_len = (
                 len(lines[0].rstrip()) - len(dedented_lines[0].rstrip())
+                if len(dedented_lines) > 0
+                else len(lines[0].rstrip())
                 if len(lines) > 0
                 else 0
```

Comment:
This condition is trying to calculate how many characters to remove from the beginning of each line to deindent it. `len(lines[0].rstrip())` is almost certainly the wrong answer; we don't want to remove 40 characters from every line if the first line is 40 characters long. Instead, we should just not remove any characters in this case.

```suggestion
                if len(dedented_lines) > 0 and len(lines) > 0
                else 0
```